docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -219,7 +219,9 @@ relation to port exhaustion.
 
 ## Indexer (`indexer`)
 
-- `ISSUE_INDEXER_TYPE`: **bleve**: Issue indexer type, currently support: bleve or db, if it's db, below issue indexer item will be invalid.
+- `ISSUE_INDEXER_TYPE`: **bleve**: Issue indexer type, currently support: bleve, db or elasticsearch, if it's db, below issue indexer item will be invalid.
```

Comment:
```suggestion
- `ISSUE_INDEXER_TYPE`: **bleve**: Issue indexer type, currently supported: `bleve`, `db` or `elasticsearch`.
```
This sentence `if it's db, below issue indexer item will be invalid` is confusing. It's better to specify which items are valid on the items themselves, as done in `app.ini.sample`.